torchbenchmark/models/sam/__init__.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# This software may be used and distributed according to the terms of the GNU General Public License version 3.
+
+from ...util.model import BenchmarkModel
+from sam import Sam
+from image_encoder import ImageEncoderViT
+from mask_decoder import MaskDecoder
+from prompt_encoder import PromptEncoder
+from transformer import TwoWayTransformer
+from predictor import SamPredictor
+
+
+from torchbenchmark.tasks import ComputerVision
+import torch
+
+    
+class Model(BenchmarkModel):
+    task = ComputerVision.SEGMENTATION
+    DEFAULT_EVAL_BSIZE = 32
+    
+    def __init__(self, test, device, jit=False, batch_size=1, extra_args=[]):
+        super().__init__(test=test, device=device, jit=jit, batch_size=batch_size, extra_args=extra_args)
+        
+        # Checkpoint options are here https://github.com/facebookresearch/segment-anything#model-checkpoints
+        sam_checkpoint = "sam_vit_h_4b8939.pth"
```

Comment:
Let's load the checkpoint from the `.data/` subdirectory under this model directory.

Reply:
Same thing for datasets like images?